src/main/java/net/fabricmc/loom/api/mappings/layered/MappingsNamespace.java

```diff
@@ -60,6 +70,8 @@ public enum MappingsNamespace {
 	public static @Nullable MappingsNamespace of(String namespace) {
 		return switch (namespace) {
 		case "official" -> OFFICIAL;
+		case "client" -> CLIENT;
+		case "server" -> SERVER;
```

Comment:
I think this should be `officialClient` and `officialServer`

Reply:
I used `client` and `server` for the sake of compatibility with existing Babric intermediaries. Would that be worth it or should I change it anyway?

Reply:
Change it, you can trivially rename those when you make your IntermediaryMappingsProvider.